qiskit_aer/backends/aer_simulator.py

```diff
@@ -730,6 +731,13 @@ def configuration(self):
         config.backend_name = self.name()
         return config
 
+    def _execute_direct(self, circuits, noise_model, config):
+        """Execute circuits on the backend.
+        """
+        aer_circuits = generate_aer_circuits(circuits)
+        ret = cpp_execute_direct(self._controller, aer_circuits, noise_model, config)
```

Comment:
Maybe call this `cpp_execute_aer_circuit` instead of direct? Not sure, but I've been getting a bit confused with some of the function naming. Like execute vs run and now the direct variants. It's getting a bit hard for me to follow because we have at least 5 things now named execute or run in these classes.

Reply:
I agree with you :-)

I would like to rename methods as follows:
```
run() --> _run_qobj()     -> [ThreadPool] -> __execute_qobj()     -> _execute_qobj()
      --> _run_circuits() -> [ThreadPool] -> __execute_circuits() -> _execute_circuits()
```